react/the_react_ecosystem/fetching_data_in_react.md

```diff
@@ -201,41 +203,41 @@ If we ever needed to fetch images in different components, instead of rewriting
+To see this in action, go back to that embedded CodeSandbox and comment out the current `Profile` and `Bio` components, and uncomment the currently commented ones.
 
-Now we have both requests firing as soon as `Profile` renders. The request for `imageURL` resolves 2 seconds before the `bioText` request and our div containing `<Bio />` renders. When `bioText` resolves an update will be made in state which will trigger a rerender in `<Bio />`, adding that text description to the page.
+Now we have both requests firing as soon as `Profile` renders. The request for `imageURL` resolves 2 seconds before the `bioText` request, and our div containing `<Bio />` renders. When `bioText` resolves, an update will be made in state which will trigger a rerender in `<Bio />`, adding that text description to the page.
 
 <div class="lesson-note lesson-note--warning" markdown="1" >
 
-In all of the code examples above, we added an artificial `delay` with the `setTimeout` function. You've guessed it by now, but it's to help you walkthrough the data fetching basics in the lesson. We recommend you remove the `delay`s and play around with the code examples once more to cement the concepts.
+In all of the code examples above, we added an artificial `delay` with the `setTimeout` function. As you've likely guessed by now, this is to help you walk through the data fetching basics in the lesson. We recommend removing these `delay`s and play around with the code examples to further cement the concepts.
 
 </div>
 
 ### Data fetching libraries
 
-We haven't even begun to scratch the surface of data fetching in the frontend. Keeping your frontend data most up-to-date with the server is a difficult task to accomplish. Managing "async" state becomes harder with every new feature. 
+We've only just begun to scratch the surface of data fetching on the frontend. Keeping your frontend data up-to-date with the server is a challenging task to accomplish. Managing "async" state becomes increasingly complex with each added feature.
 
-You've already tasted the complexity of data fetching in this lesson. Each request has to have a _minimum_ of three states to achieve an optimal user experience: `data`, `loading`, and `error`. There's a library called [TanStack Query](https://tanstack.com/query/latest/docs/react/overview) that helps you in data fetching and more. It keeps track of all the necessary states for you, and has built-in support for major features like caching.
+You've already tasted the complexity of data fetching in this lesson. Each request has to have a _minimum_ of three states to achieve an optimal user experience: `data`, `loading`, and `error`. There's a library called [TanStack Query](https://tanstack.com/query/latest/docs/react/overview) that helps you with data fetching and more. It keeps track of all the necessary states for you and has built-in support for major features like caching.
```

Comment:
Since this link is in Additional Resources, we should remove it from here as it's out of place. This and the following line will need to be updated a bit; maybe instead we just mention something along the lines of "There are libraries that can help with data fetching and more, but we strongly suggest.... rest of line 232 here..."